articles/virtual-machines/windows/hibernate-resume-windows.md

```diff
@@ -151,13 +148,89 @@ To learn more about REST, check out an [API example](/rest/api/compute/virtual-m
 
 ---
 
-Once you've created a VM with hibernation enabled, you need to configure the guest OS to successfully hibernate your VM. 
+Once you've created a VM with hibernation enabled, you need to configure the guest OS to successfully hibernate your VM.
+
+## Enabling hibernation on an existing Windows VM 
+
+To enable hibernation on an existing VM, you can use Azure CLI, PowerShell and API. Before proceeding, ensure that the guest OS version supports hibernation on Azure. Refer to the list of [supported OS versions ](../hibernate-resume-windows.md#supported-windows-versions).
```

Comment:
Remove ../ from link. Should present as:

`(hibernate-resume-windows.md#supported-windows-versions)`

Reply:
Fixed